src/routes/data-decode/__tests__/data-decode.e2e-spec.ts

```diff
@@ -55,6 +56,40 @@ describe('Data decode e2e tests', () => {
       });
   });
 
+  it('POST /data-decoder should throw a validation error', async () => {
```

Comment:
Moving the validation checks here since these are performed at controller/validation pipe level now.